BlazarUI/app/stylus/type.styl

```diff
@@ -47,4 +47,7 @@ td
 
 .notfound__heading
   font-weight 100
-  margin-bottom 20px
\ No newline at end of file
+  margin-bottom 20px
+
+.headline-no-breadcrumbs
```

Comment:
move into headline component's styl file since this is specifically addressing that component. Also, try not to resort to using !important when it's not necessary, it causes much more trouble than good in the long run, just try to be [more specific](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) - e.g.

```stylus
.headline
  font-size 26px
  margin-top 8px
  margin-bottom 0
  padding 5px 0 10px 0

  &.headline-no-breadcrumbs
    margin-top 35px
```